README.md

```diff
@@ -97,6 +97,13 @@ To build with lua instead of luaJIT, run:
 $ ./x.py build -DENABLE_LUAJIT=OFF
 ```
 
+To build with debug symbols, run:
```

Comment:
It's not correct. `RelWithDebInfo` also includes debug symbols.

Reply:
I tested that the default compilation option calng had no way of single-line debugging of the break point.ubuntu 22

Reply:
I think this line is a bit misleading. For `RelWithDebInfo` it did bring the debug information but the optimization level is `O2`. So guess the issue you described should be caused by the compiler optimization instead of the debug information.

Reply:
That's the idea.